lib/application.js

```diff
@@ -17,6 +17,7 @@ var extend = require('util')._extend;
 var RemoteObjects = require('strong-remoting');
 var classify = require('underscore.string/classify');
 var camelize = require('underscore.string/camelize');
+var configureSharedMethods = require('./configure-shared-methods');
```

Comment:
This `require()` is no longer needed, let's remove it. (I should have removed it myself, sorry!)

Reply:
done